ci-operator/config/apache/camel-k/apache-camel-k-main__camelk-ocp4.13-lp-interop.yaml

```diff
@@ -0,0 +1,43 @@
+base_images:
+  cli:
+    name: "4.13"
+    namespace: ocp
+    tag: cli
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.19
+releases:
+  latest:
+    candidate:
+      product: ocp
+      stream: nightly
+      version: "4.13"
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: camelk-amp-interop-aws
+  cron: 0 1 * * 1
+  steps:
+    cluster_profile: aws-cspi-qe
+    env:
+      BASE_DOMAIN: aws.interop.ccitredhat.com
```

Comment:
please update BASE_DOMAIN to `cspilp.interop.ccitredhat.com` that will resolve another error you are getting:

> failed to fetch Common Manifests: failed to fetch dependency of "Common Manifests": failed to generate asset "DNS Config": getting public zone for "aws.interop.ccitredhat.com": No public route53 zone found matching name "aws.interop.ccitredhat.com"

Reply:
changed